_posts/2023-7-12-Add-client-side-HTTP-authentication-mechanism-support.adoc

```diff
@@ -0,0 +1,129 @@
+---
```

Comment:
Just for consistency with other posts, please use double digits for the month in the file name, i.e., 07.